### PR TITLE
Allow sliderbars to have delayed value transfers

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: osu.Framework/Graphics/UserInterface/SliderBar.cs [hunk 1]
```diff
@@ -141,6 +181,18 @@ protected override bool OnKeyDown(KeyDownEvent e)
             }
         }
 
+        protected override bool OnKeyUp(KeyUpEvent e)
+        {
+            commit();
```
Comment (ekrctb) [inline osu.Framework/Graphics/UserInterface/SliderBar.cs]: Values are committed when an irrelevant keys is unpressed, right?

File: osu.Framework/Graphics/UserInterface/SliderBar.cs [hunk 2]
```diff
@@ -141,6 +181,18 @@ protected override bool OnKeyDown(KeyDownEvent e)
...
+            return base.OnKeyUp(e);
```
Comment (UselessToucan) [inline osu.Framework/Graphics/UserInterface/SliderBar.cs]: Are you sure that it should not be `return true;` here? The keyboard input was handled.

Comment (peppy) [reply osu.Framework/Graphics/UserInterface/SliderBar.cs]: not really important either way

File: osu.Framework/Graphics/UserInterface/SliderBar.cs (hunk 1 above)
Comment (peppy) [reply osu.Framework/Graphics/UserInterface/SliderBar.cs]: yeah, can probably guard this for safety (will be required with a refactor i'm currently making).

Comment (peppy) [reply osu.Framework/Graphics/UserInterface/SliderBar.cs]: see latest commit; changes logic quite a bit as it did not work in one specific scenario.

File: osu.Framework/Configuration/Bindable.cs [hunk 3]
```diff
@@ -278,6 +278,16 @@ internal void Reset()
 
         IBindable<T> IBindable<T>.GetBoundCopy() => GetBoundCopy();
 
+        /// <summary>
+        /// Create an unbound clone of this bindable.
+        /// </summary>
+        public IBindable<T> Clone()
```
Comment (ekrctb) [inline osu.Framework/Configuration/Bindable.cs]: Name this `GetUnboundCopy` or something like that?

Comment (peppy) [reply osu.Framework/Configuration/Bindable.cs]: That may suit better. @smoogipoo thoughts?

**Pull request merged**